scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -260,6 +260,22 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
                             "and $scannerCriteria, not scanning the package again $packageIndex."
                 }
 
+                val packagePath = pkg.id.toPath()
```

Comment:
I read through the commit message again and propose the following changes:

> LocalScanner: Check for archived files if scan results are present

This does not only check for presence, but also recreate the archives if needed. So I'd change this to "LocalScanner: (Re-)Create license file archives if needed"

The body text I'd reword as follows:

The LocalScanner supports both an archive (for license files) and a
storage (for scan results) for a package. So far, if scan results are
present in the storage, scanning is skipped independently of the presence
of an archive. Allow to (re-)create the archive in such cases so reporters
depending on the archive work properly.